### PR TITLE
Update to use config-free tracking

### PR DESCRIPTION
**Pull request by joshuarrrr**
Note that we're using this opportunity to automatically migrate most of our customers, so no optional support of the old method.

### CODE REVIEW

File: tests/test-wp-parsely.php [hunk 1]
```diff
@@ -193,24 +193,7 @@ public function setUp() {
 		);
 		update_option( 'parsely', $option_defaults );
 		self::$parsely_html = <<<PARSELYJS
-<div id="parsely-root" style="display: none">
-	<div id="parsely-cfg" data-parsely-site="blog.parsely.com"></div>
-</div>
-<script data-cfasync="false">
```
Comment (hbbtstar) [inline tests/test-wp-parsely.php]: we need to keep the data-cfasync thing, or cloudfront will kablammo our script

File: parsely-javascript.php [hunk 2]
```diff
@@ -54,23 +54,6 @@ function uuidProfileCall() {
 	</script>
 <?php endif; ?>
 
-<div id="parsely-root" style="display: none">
-	<div id="parsely-cfg" data-parsely-site="<?php echo esc_attr( $parsely_options['apikey'] ); ?>"></div>
-</div>
-<script data-cfasync="false">
```
Comment (hbbtstar) [inline parsely-javascript.php]: same comment as in the test re: data-cfasync

File: tests/test-wp-parsely.php (hunk 1 above)
Comment (joshuarrrr) [reply tests/test-wp-parsely.php]: ah, totally missed that, thanks for the catch.

**Comment by joshuarrrr** [comment]
@hbbtstar can you take a final look to make sure I did what you meant?

**Pull request merged**